rules/jsxExpressionSpacingRule.ts

```diff
@@ -28,7 +28,10 @@ class JsxExpressionSpacingWalker extends Lint.RuleWalker {
 				this.createFailure(
 					node.getStart() + 1,
 					1,
-					'jsx expression should have one space after opening \'{\''
+					'jsx expression should have one space after opening \'{\'',
+					new Lint.Fix('jsx-expression-spacing', [
+						this.appendText(node.getStart() + 1, ' ')
```

Comment:
It looks like this only addresses `{value}`. What about extra whitespace? Could make autofix annoying if it keeps adding more spaces when it should be deleting them